pkg/agent/datamodel/types.go

```diff
@@ -2106,3 +2107,26 @@ func (a *AgentPoolProfile) IsDisableWindowsOutboundNat() bool {
 		a.AgentPoolWindowsProfile.DisableOutboundNat != nil &&
 		*a.AgentPoolWindowsProfile.DisableOutboundNat
 }
+
+// SecurityProfile begin.
+type SecurityProfile struct {
+	PrivateEgress *PrivateEgress `json:"privateEgress,omitempty"`
+}
+
+type PrivateEgress struct {
+	Enabled      bool   `json:"enabled"`
+	ProxyAddress string `json:"proxyAddress"`
```

Comment:
Will it be possible to support multiple addresses?

Reply:
the proxy is managed by AKS. for current design, it's always single.

Reply:
@karataliu  Could it be possible to support multiple addresses in future? We hit similar issues in other features and we have to add a new property later. It is better to use a string array to avoid such issue. How do you think?

Reply:
this is service managed address, so suppose won't be multiple.